submissions/pozzitive11/memoty-pair-game/js/index.js

```diff
@@ -0,0 +1,127 @@
+const cards = [
+  {
+    front: "images/front.jpg",
+    back: "images/sleepy-cat.jpg",
+    name: "sleepy",
+  },
+  {
+    front: "images/front.jpg",
+    back: "images/cat.jpg",
+    name: "cat",
+  },
+  {
+    front: "images/front.jpg",
+    back: "images/frightened-cat.jpg",
+    name: "frightened",
+  },
+  {
+    front: "images/front.jpg",
+    back: "images/brutal-cat.jpg",
+    name: "brutal",
+  },
+  {
+    front: "images/front.jpg",
+    back: "images/rock-cat.jpg",
+    name: "rock",
+  },
+  {
+    front: "images/front.jpg",
+    back: "images/cute-cat.jpg",
+    name: "cute",
+  },
+];
+
+const createCard = (front, back, name) => {
+  return `<div class="game__card" id="${name}">
+            <img class="front__img" src="${front}" alt="">
+            <img class="back__img" src="${back}" alt="">
+          </div>
+          <div class="game__card" id="${name}">
+            <img class="front__img" src="${front}" alt="">
+            <img class="back__img" src="${back}" alt="">
+          </div>`;
+};
+
+const gameCard = cards.map(({ front, back, name }) =>
+  createCard(front, back, name)
+);
+const gameContainer = document.querySelector(".game__container");
+gameContainer.innerHTML = gameCard.join("");
+const gameCards = document.querySelectorAll(".game__card");
+
+let counter = 0;
+let arr = [];
+
+function flipCard() {
+  if (this && this.classList.contains("game__card")) {
+    this.classList.add("flip");
+
+    arr.push(this);
+  }
+  checkArrLength();
+
+  winCOndition();
```

Comment:
```suggestion
  winCondition();
```

Also, please change function name. It should be verb